server/graphql/v2/interface/Transaction.js

```diff
@@ -113,6 +119,12 @@ export const Transaction = new GraphQLInterfaceType({
       permissions: {
         type: TransactionPermissions,
       },
+      isRejected: {
+        type: GraphQLBoolean,
```

Comment:
Should match the one below

```suggestion
        type: new GraphQLNonNull(GraphQLBoolean),
```